docs/source/models/cause.rst

```diff
@@ -41,29 +41,251 @@ Common data sources for cause models
 
 .. todo::
 
-   Format as table with measure, measure definition, data sources and
-   their uses.
+   Update mortality-related data sources within existing format.
 
-Incidence
-+++++++++
+Once a cause model structure is specified, data is needed to inform its states and transitions. For our purposes, cause models generally have the following data needs:
 
-Birth prevalence
-++++++++++++++++
+1. The probability that a simulant will start the simulation in a given state within the cause model
+2. The probability that a simulant will transition to a new state within the cause model in a given timestep
+3. The disability weight for each state in the cause model
+4. The probability that a simulant in a given cause model state will die in a given timestep
+
+There are several common data sources that can be used for these needs, which are outlined in the table below and discussed in more detail afterward.
 
-Remission
-+++++++++
++--------------+--------------------------------+--------------------+
+|Measure       |Definition                      |Uses                |
++==============+================================+====================+
+|Prevalence    |Proportion of population        |Initialize cause    |
+|              |with a given condition          |model states        |
++--------------+--------------------------------+--------------------+
+|Birth         |Proportion of all live births   |Initialize neonatal |
+|Prevalence    |born with a given condition     |cause model states  |
++--------------+--------------------------------+--------------------+
+|Incidence     |Number of new cases of a given  |Estimate transition |
+|              |condition per person-year       |rates               |
++--------------+--------------------------------+--------------------+
+|Remission     |Number of recovered cases from a|Estimate transition |
+|              |given condition per person-year |rates               |
++--------------+--------------------------------+--------------------+
+|Disability    |INSERT DEFINITION (Yaqi?)       |INSERT USES (Yaqi?) |
+|Weights       |                                |                    |
++--------------+--------------------------------+--------------------+
+|Cause-Specific|INSERT DEFINITION (Yaqi?)       |INSERT USES (Yaqi?) |
+|Mortality     |                                |                    |
++--------------+--------------------------------+--------------------+
+|Excess        |INSERT DEFINITION (Yaqi?)       |INSERT USES (Yaqi?) |
+|Mortality     |                                |                    |
++--------------+--------------------------------+--------------------+
 
 Prevalence
```

Comment:
It is possible to add hyperlinks between the entries and headings, but I don't remember how to do it.

Reply:
Done!